LayoutTests/http/tests/privateClickMeasurement/database-disabled-in-ephemeral-session.html

```diff
@@ -8,7 +8,7 @@
 </head>
 <body onload="setTimeout(runTest, 0)">
 <div id="description">Tests that stored attributions cannot be converted in an ephemeral session.</div>
```

Comment:
This needs to be changed too. "… doesn't use the data base in …" That'll effect text output, I think.